File: src/ProjectR/content/scripts/spells/HiddenSeal37.lua
function GetTargetType()
    return Decaying
end

function GetName()
  return "Hidden Seal #37"
end

function GetDescription()
    return "Something unknown surges through your enemies.\nDeals damage based on proximity to target."
end

function IsSupportSpell()
    return false
end

function GetMPCost()
    return .66
end

function GetDelay()
  return .45
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return {  }
end

function SpellEffect()
  local damage = (4 * aMD - dMR) / modifier
  d:TakeDamage(damage)
end

File: src/ProjectR.Scripting/content/scripts/spells/HiddenSeal37.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting.Spells
{
    public class HiddenSeal37 : SpellScriptBase
    {
        public override string Name { get { return "Hidden Seal #37"; } }
        public override string Description { get { return "Something unknown surges through your enemies.\nDeals damage based on proximity to target."; } }

        public override TargetType TargetType { get { return TargetType.Decaying; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return false; } }

        public override double MPCost { get { return 66; } }
        public override double Delay { get { return .45; } }

        public override void SpellEffect(ICharacter caster, ICharacter target, double modifier)
        {
            var damage = (4 * cMD - tMR) / modifier;
            DealDamage(damage);
        }
    }
}